recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "2.2.0" %}
+{% set version = "2.3.0" %}
 
 package:
   name: pre-commit
   version: {{ version }}
 
 source:
   url: https://github.com/pre-commit/pre-commit/archive/v{{ version }}.tar.gz
-  sha256: 53a5d39e8b2063a004ecdabd4b459ae826cfe47eca449720e4fdde06a7d43cc0
+  sha256: cbe534c57d0e0e17b0a739d71bb0d33a3bc295e65d6d125860bcfc7053abd645
 
 build:
-  number: 1
+  number: 0
   skip: true  # [py<36]
```

Comment:
maybe we can remove this and set this package as noarch

of course we don't need to do it in this PR

any thought?

Reply:
Agree. 👍

Reply:
https://github.com/conda-forge/pre-commit-feedstock/pull/18#issuecomment-619540601